File: Backend/TravelPlanner.Core/Triposo/CityWalk.cs
using Newtonsoft.Json;

namespace TravelPlanner.Core.Triposo
{
    public class CityWalk
    {
        [JsonProperty("seed")]
        public int Seed { get; set; }

        [JsonProperty("total_duration")]
        public int TotalDuration { get; set; }

        [JsonProperty("walk_distance")]
        public int WalkDistance { get; set; }

        [JsonProperty("walk_duration")]
        public int WalkDuration { get; set; }

        [JsonProperty("way_points")]
        public WayPoint[] WayPoints { get; set; }
    }
}

File: TravelPlannerApp/app/src/main/java/com/travelplanner/models/CityWalk.kt
package com.travelplanner.models

class CityWalk(val cityWalkId: String, val seed: Int, val totalDuration: Int, val walkDuration: Int, val walkDistance: Int, val wayPoints: List<WayPoint>)

class WayPoint(val wayPointId: String, val latitude: Float, val longitude: Float, val poi: Poi, val visitTime: Int, val walkToNextDistance: Int, val walkToNextDuration: Int)